ui/app/modifiers/autofocus.js

```diff
@@ -0,0 +1,7 @@
+import { modifier } from 'ember-modifier';
+
+export default modifier(function autofocus(element, _positional, named) {
+  const { ignore } = named;
+  if (ignore) return;
+  element.focus();
+});
```

Comment:
`praise`: Great job making this modifier.

`side-note`: Tons of great addons exist in the Ember ecosystem. This one from my friends at Qonto is pretty good for handling `focus` with child elements in addition to the parent element:  https://github.com/qonto/ember-autofocus-modifier